routes/frontend/web.php

```diff
@@ -208,4 +209,6 @@
                 return $media;
             })->name('media.private');
         });
+
+        Route::get('/order-public/{order:uuid}', PublicLink::class)->name('order.public');
```

Comment:
as this should be protected by the signature you should move this to the block where the signed middleware is used